.github/workflows/sdk.yaml

```diff
@@ -94,7 +94,7 @@ jobs:
             cargo "${sub}" --package google-cloud-wkt --no-default-features --features time
             cargo "${sub}" --package google-cloud-wkt --all-features
           done
-          cargo clippy --no-deps --package google-cloud-wkt --all-features --profile=test -- --deny warnings
+          cargo clippy --no-deps --package google-cloud-wkt --all-features --all-targets --profile=test -- --deny warnings
 
           cargo clean
           echo "==== google-cloud-gax ===="
```

Comment:
Unrelated aside: should we split the different packages into different steps of the build?

Reply:
We could, but then we need to add requirements to GitHub pull requests when packages that have features change.